packages/strapi-parts/src/Dialog/Dialog.stories.mdx

```diff
@@ -18,7 +18,7 @@ import DeleteIcon from '@strapi/icons/DeleteIcon';
 
 # Dialog
 
-This is the doc of the `Dialog` component
+`Dialog` creates a box to confirm a delete action.
```

Comment:
It's not always a delete action. In most of the case yes, but I'd then remove the word "delete"